File: src/main/java/akostenko/aicars/race/cartest/CircularCruiseTest.java
package akostenko.aicars.race.cartest;

import akostenko.aicars.model.CarModel;
import akostenko.aicars.race.Driver;

public class CircularCruiseTest extends Driver {
    @Override
    public double accelerating() {
        return 1;
    }

    @Override
    public double breaking() {
        return 0;
    }

    @Override
    public double steering() {
        return CarModel.peakLateralForceAngle;
    }

    @Override
    public String getName() {
        return "Max speed turning";
    }

    @Override
    public void update(double dTime) {

    }
}

File: src/main/kotlin/akostenko/aicars/race/cartest/CircularCruiseTest.kt
package akostenko.aicars.race.cartest

import akostenko.aicars.model.CarModel
import akostenko.aicars.race.Driver

class CircularCruiseTest : Driver() {
    override fun accelerating(): Double {
        return 1.0
    }

    override fun breaking(): Double {
        return 0.0
    }

    override fun steering(): Double {
        return CarModel.peakLateralForceAngle
    }

    override val name: String
        get() = "Max speed turning"

    override fun update(dTime: Double) {

    }
}
